src/flb_dlfcn_win32.c

```diff
@@ -30,16 +29,16 @@ static void store_error(void)
         return;
     }
 
-    if (!FormatMessage(FORMAT_MESSAGE_ALLOCATE_BUFFER |
-                       FORMAT_MESSAGE_FROM_SYSTEM |
-                       FORMAT_MESSAGE_IGNORE_INSERTS,
-                       NULL,
-                       err,
-                       MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT),
-                       (LPTSTR) &errbuf,
-                       _countof(errbuf), NULL))
-        errbuf[0] = '\0';
-    currerr = errbuf;
+    if (!FormatMessageA(FORMAT_MESSAGE_FROM_SYSTEM |
+                        FORMAT_MESSAGE_IGNORE_INSERTS,
+                        NULL,
+                        err,
+                        MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT),
+                        (LPTSTR) &dlerrorbuf,
```

Comment:
`FormatMessageA()`  expects `char*` for the 5th argument but it passes `char**`.
Probably it causes crash.

https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-formatmessagea

In addition you shouldn't cast the type forcedly, it squashes such compiler warnings errors.